### DISCUSSION TITLE
A Second Google-Chat Run.

### DISCUSSION THREAD

**Discussion by Atlaimond** [ankurk91/google-chat-electron#115] [Ideas]
System: **Linux Mint 20.3 Xfce** (up to dated) 
Additional PPA: **libreoffice-still** 
Additional repositories: **Protonvpn / Stable** 
 
Hello. 👋️ 
I would like to be able to launch or run a second [Google-Chat] application of yours. My goal is to be able to be logged-in both my real-life account and my casual-account in the same time. 
I know I can easily swap accounts, but the other account goes "_Away_", as I do this. 
 
If it is already possible to achieve this in [Linux Mint 20.2 Xfce], please point me to a "_programmer non-expert_" level of guide. 
 
PS. I've already found a temporary workaround, I log-in, with my casual account. via [Firefox] web-browser, while I log-in, my real-life account, via your [Google-Chat] application! 🤓️💡️ 
Thank you for whatever reply you'll give me. 🙇‍♂️️

**Comment by ankurk91** [comment]
Sorry, 2 instances of the application won't work, it was never designed in that way.

**Reply by Atlaimond** [reply]
Aww. 🙁️ Okay, thanks. It was worth to try. 🤷‍♂️️

**Comment by ankurk91** [comment]
If you are a developer, you can try to remove this limitation.

https://github.com/ankurk91/google-chat-electron/blob/234777a04aa67e595ffbe6992bf1ba1e8373efea/src/main/index.ts#L5

I don't know the side effects, best of luck.

Electron doesn't force this limitation, I have manually added this limit.
